backend/spec/export_ead_spec.rb

```diff
@@ -1515,21 +1498,32 @@ def get_xml_doc
                                                          :suppressed => true)
 
           @xml = get_xml_doc
+          raise Sequel::Rollback
         end
       end
     }
 
-    after(:all) do
-      as_test_user('admin') do
-        $repo_id = $old_repo_id
-        JSONModel.set_repository($repo_id)
-      end
-    end
-
     it "excludes suppressed items" do
       expect(@xml.xpath('//c').length).to eq(2)
       expect(@xml.xpath("//c[@id='aspace_#{@unsuppressed_series.ref_id}']").length).to eq(1)
       expect(@xml.xpath("//c[@id='aspace_#{@unsuppressed_series_unsuppressed_child.ref_id}']").length).to eq(1)
     end
   end
+
+  # See ANW-1282
+  describe "Metadata Rights Declaration mappings " do
+    it "maps all subrecords to ead/control/filedesc/publicationstmt" do
```

Comment:
Just confirming that it is desired that all information from the rights declaration should be exported in EAD2002?  From reading the latest comment on the JIRA I assumed that only the text of the <descriptivenote> was intended to be exported inside a <p> inside of <publicationstmt>.

Reply:
I'm going to go with your interpretation. Added a new commit for this.